odl/space/ntuples.py

```diff
@@ -1320,7 +1349,7 @@ def adjoint(self):
                                       ''.format(self.domain.field,
                                                 self.range.field))
         return MatVecOperator(self.range, self.domain,
-                              np.asarray(np.asmatrix(self.matrix).H))
+                              self.matrix.conj().T)
```

Comment:
I think we should change `self.matrix` to `self.array` simply to stop the implication that this is a matrix.